lib/checks/color/link-in-text-block.json

```diff
@@ -5,10 +5,14 @@
     "impact": "serious",
     "messages": {
       "pass": "Links can be distinguished from surrounding text in some way other than by color",
-      "fail": "Links need to be distinguished from surrounding text in some way other than by color",
+      "fail": {
+        "default": "Links need to be distinguished from surrounding text in some way other than by color",
```

Comment:
Doesn't look like we can ever really hit this code path still. The only return false that gets to this is here: https://github.com/dequelabs/axe-core/pull/3575/files#diff-ca029eef8fd1995a0339c5f17452abc9b469d1f874a12d01ad89a2d1f42d819bR30 and that one shouldn't be hit either, since the matches should prevent non-block nodes from getting into the check.

Reply:
Remove the default case for failure